wallop.module

```diff
@@ -72,48 +72,10 @@ function wallop_add_core_assets() {
     return;
   }
 
-  $assets = wallop_read_core_assets();
-
-  drupal_alter('wallop_asset', $assets);
-
-  drupal_add_js($assets['js']['asset'], $assets['js']['args']);
-  drupal_add_css($assets['css']['asset'], $assets['css']['args']);
+  $base_path = drupal_get_path('module', 'wallop');
+  drupal_add_js($base_path . '/js/vendor/wallop.min.js');
+  drupal_add_js($base_path . '/js/wallop.js');
+  drupal_add_css($base_path . '/css/wallop.css');
```

Comment:
Any reason to move these out of their own function? Was somewhat nice to have the `drupal_alter` in case we ever wanted to inject our own JS/CSS assets for a slideshow.